src/App.js

```diff
@@ -1,16 +1,52 @@
 import React from 'react';
-
-class App extends React.Component {
+import TodoItems from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
-  }
+  class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+        listData: [{ },],
```

Comment:
it can sometimes be easier to start with some example elements to keep things straight, they give some examples you could use in the readme